cli/script/command-parser.ts

```diff
@@ -116,11 +136,12 @@ function deploymentRemove(commandName: string, yargs: yargs.Argv): void {
 
 function deploymentHistory(commandName: string, yargs: yargs.Argv): void {
     isValidCommand = true;
-    yargs.usage(USAGE_PREFIX + " deployment " + commandName + " <appName> <deploymentName> [--format <format>]")
+    yargs.usage(USAGE_PREFIX + " deployment " + commandName + " <appName> <deploymentName> [--format <format>] [--displayReleasedBy]")
         .demand(/*count*/ 4, /*max*/ 4)  // Require exactly four non-option arguments.
         .example("deployment " + commandName + " MyApp MyDeployment", "Shows the release history for deployment \"MyDeployment\" from app \"MyApp\" in tabular format")
         .example("deployment " + commandName + " MyApp MyDeployment --format json", "Shows the release history for deployment \"MyDeployment\" from app \"MyApp\" in JSON format")
-        .option("format", { default: "table", demand: false, description: "The output format (\"json\" or \"table\")", type: "string" });
+        .option("format", { default: "table", demand: false, description: "The output format (\"json\" or \"table\")", type: "string" })
+        .option("displayReleasedBy", { alias: "rb", default: false, demand: false, description: "Whether to display the released by email address", type: "boolean" });
```

Comment:
I think its a given that the corresponding the tabular format represents `who performed the release`, instead of `each release`. Will go with Damon's suggestion.